webvtt.html

```diff
@@ -542,6 +542,17 @@
    </section>
 
    <section>
+    <h3>Unicode normalization</h3>
+
+    <p>This specification does not normalize Unicode text as part of its processing
+    requirements.</p>
+
+    <p>For example, a cue with the identifier consisting of the LATIN CAPITAL LETTER A followed by
```

Comment:
> On Jun 9, 2015, at 12:07 , Philip Jägenstedt <notifications@github.com> wrote:
> 
> In webvtt.html:
> 
> > @@ -542,6 +542,17 @@
> >     </section>
> >  
> >     <section>
> > +    <h3>Unicode normalization</h3>
> > +
> > +    <p>This specification does not normalize Unicode text as part of its processing
> > +    requirements.</p>
> > +
> > +    <p>For example, a cue with the identifier consisting of the LATIN CAPITAL LETTER A followed by
> 
> Normalization isn't optional, it's not part of the processing requirements and thus forbidden. In other words, there's no interop issue here if people implement what the spec says.

Then let’s say that unequivocally;  WebVTT text must not be normalized.

change "This specification does not normalize Unicode text as part of its processing requirements.”

to “This specification requires that Unicode text not be normalized.”

(The previous statement could be read by a trouble-maker — or someone who has a text engine that always normalizes — as allowing but not requiring normalization.)

David Singer
Manager, Software Standards, Apple Inc.